KeyboardVisualizerVC/KeyboardVisualizerVC.vcxproj

```diff
@@ -182,6 +182,7 @@
     <ClCompile Include="..\KeyboardVisualizerCommon\LogitechSDK.cpp" />
     <ClCompile Include="..\KeyboardVisualizerCommon\MSIKeyboard.cpp" />
     <ClCompile Include="..\KeyboardVisualizerCommon\net_port.cpp" />
+    <ClCompile Include="..\KeyboardVisualizerCommon\PoseidonZRGBKeyboard.cpp" />
```

Comment:
Just thought could be nice cause the project can't be build unless you manually add the poseidonz files to the project.

Regarding the other things. I will update my code :)